### DISCUSSION TITLE
Failed to access data from shortcut

### DISCUSSION THREAD

**Discussion by Zhou-Shilin** [utmapp/UTM#4969] [UTM Help (iOS)]
When importing *.utm file created from another user, there is an error message saying "Failed to access data from shortcut.".

iPadOS 16.2 Beta 2, using Jitterbug + Sideloader
iPad Air 5 (M1)

**Comment by ktprograms** [comment]
How did the other user export the VM? Was it done with the "Move" option or the "Share" option?
